### DISCUSSION TITLE
[预览]v1.12.5

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#1752] [分享 / Share]
- 新增功能`传统连播模式`. (#1742)
> 使用传统的连播模式, 视频有多P时 / 在收藏夹或稍后再看列表里时自动开启连播, 单P视频自动关闭连播防止播放推荐视频.
> 并非是切断多P连播和推荐连播的绑定, 所以我也不敢保证这功能一定靠谱(

- 附加功能中的直播勋章更换支持显示已变灰的勋章. (#873)
- 修复直播间相关功能失效. (#1722)
- 修复`直播全屏包裹`弹窗的小箭头位置.
- `删除视频弹窗`功能分为更细致的3项: 关注 / 投票 / 关联视频, 可以分别设置是否屏蔽. (#1695)

(Release 好像能附带一个 discussion 了, 我开一个试试

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.5-preview'>[预览]v1.12.5</a>.</em>

**Comment by VergilGao** [comment]
我现在已经习惯手动换集了……TAT
